kv/txn.go

```diff
@@ -32,6 +32,7 @@ func RunInNewTxn(store Storage, retryable bool, f func(txn Transaction) error) e
 		txn           Transaction
 	)
 	for i := uint(0); i < maxRetryCnt; i++ {
+		store.SetPriority(PriorityLow)
```

Comment:
Setting priority here will affect other operations that uses RunInNewTxn.

Reply:
Ok, I changed.